wolox-books/src/app/screens/user/components/register/register.component.html

```diff
@@ -0,0 +1,20 @@
+<form [formGroup]="formGroup" (ngSubmit)="request(formGroup)" class="form">
+  <label for="firstname" class="label">Nombre</label>
+  <input type="text" class="input-text" formControlName="first_name">
+  <div class="error" *ngIf="!firstName.valid && firstName.touched">This field is required.</div>
+  <label for="lastname" class="label">Apellido</label>
+  <input type="text" class="input-text" formControlName="last_name">
+  <div class="error" *ngIf="!lastName.valid && lastName.touched">This field is required.</div>
+  <label for="email" class="label">Email</label>
+  <input type="text" class="input-text" formControlName="email">
+  <div class="error" *ngIf="!email.valid && email.touched">You must specify an email valid.</div>
+  <label for="password" class="label">Password</label>
+  <input type="password" class="input-text" formControlName="password">
+  <div class="error" *ngIf="!password.valid && password.touched">This field is required with at least A-Z, a-z and 0-9 characters.</div>
+  <label for="password-confirm" class="label">Confirmaci&oacute;n de Password</label>
+  <input type="password" class="input-text" formControlName="password_confirmation">
+  <div class="error" *ngIf="equalTo(formGroup) && passwordConfirmation.touched">This field is required and equal to password.</div>
```

Comment:
`equalTo` as a get accessor. Pass `formGroup` as param is needless, yo can get it by reference.

```suggestion
  <div class="error" *ngIf="equalTo && passwordConfirmation.touched">This field is required and equal to password.</div>
```